dashboard/app/helpers/levels_helper.rb

```diff
@@ -95,7 +95,7 @@ def get_channel_for(level, user = nil)
       )
     end
 
-    channel_token.try :channel
+    channel_token.channel if channel_token
```

Comment:
can also use ruby's new'ish `channel_token&.channel` syntax. any of these options are fine though

Reply:
Ah good call! I can never remember which side of the expression is allowed to be nil with the save-navigation operator.

For my own reference...

```ruby
[1] pry(main)> nil&.foo
=> nil
[2] pry(main)> 5&.foo
NoMethodError: undefined method `foo' for 5:Integer
Did you mean?  floor
from (pry):44:in `__pry__'
```